clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerNetworkClient.java

```diff
@@ -417,7 +420,7 @@ private boolean trySend(long now) {
 
     public void maybeTriggerWakeup() {
         if (!wakeupDisabled.get() && wakeup.get()) {
-            log.trace("Raising wakeup exception in response to user wakeup");
+            log.debug("Raising WakeupException in response to user wakeup");
```

Comment:
Worth mentioning in the PR description that we're changing the log level for wakeup related activity as it's in the middle of a big refactoring.